travelplan/src/SearchPage/SearchPage.css

```diff
@@ -0,0 +1,102 @@
+body {
+  margin: 0;
+  padding: 0;
+  background-color: #FFBA73;
+
+}
+
+html {
+  margin: 0;
+  padding: 0;
+}
+
+h1 {
+  display: flex;
+  font-family: 'Righteous';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 85px;
+  line-height: 106px;
+  margin-block-start: 0;
+  margin-block-end: 0;
+}
```

Comment:
Let's move it outside the component's style to the global styles or use the specific classes